helm/examples/certmanager/README.md

```diff
@@ -200,3 +208,16 @@ You should see a result similar to this:
 The default public issuer uses self signed certificates. This makes it very easy to set up but browsers
 will not recognize the certificates as trustworthy and the browser will warn users that the connection
 is not safe.
+
+How to set up a cert-manager ClusterIssuer to use something like LetsEncrypt or Zerossl is outside the
+scope of this document, but once you have one set up you can use it to generate your public certificates
+by setting the public issuer values something like this:
+
+certificates:
+  issuers:
+    public:
+      name: zerossl-issuer
+      kind: ClusterIssuer
+      spec: null
```

Comment:
Is this needed (especially since kind is ClusterIssuer)?

Reply:
Yes with this change that's now true, I'll change it.